README.md

```diff
@@ -8,7 +8,52 @@
+## Introduction
+The Media and Entertainment AI Suite is a cloud-native suite of docker containers, reference pipelines, libraries and pre-trained models that are designed to enhance video quality, improve user experience and lower costs for video service providers, such as broadcasters, video streamers, and social media service providers. These suites leverage Intel's AI libraries and frameworks, such as the Enterprise AI Framework and OpenVINO™, to deliver high-value media enhancement use cases.
+Key workloads supported include:
+- Video Super Resolution (VSR): Supports real-time or batch up-scaling of content from older pre-HD formats to HD and 4K, including support for 8-bit and 10-bit content.  VSR is available through a range of different up-scaling models, providing a trade-off between video quality and run-time performance. 
+- Video Bit Rate Optimization (SVP): Reduces video bit rate without compromising video quality, and is compatible with standard CODECs such as AVC, HEVC and AV1.  Helps to reduce costs through lower transmission and storage requirements.
+
+The libraries included in the suite are supported on both Intel® Xeon™ CPUs and Intel® Data Center GPUs, and can be easily integrated into existing workflows using FFmpeg plugins provided as part of the libraries.  The libraries are also available as part of the [Intel® Edge AI Suites](https://github.com/open-edge-platform/edge-ai-suites) platform, the [Intel® Tiber™ Broadcast Suite](https://github.com/OpenVisualCloud/Intel-Tiber-Broadcast-Suite), and the [Open Visual Cloud project](https://github.com/OpenVisualCloud/Intel-Tiber-Broadcast-Suite).
+
+### Key Features
+- **Video Super Resolution:** Includes support for four pre-trained models, optimized for best performance on Intel® Xeon™ CPUs and Intel® Datacenter GPU hardware.
+  - **Enhanced RAISR** – Optimized C/C++ implementation of the [Rapid and Accurate Image Super Resolution (RAISR)](https://arxiv.org/pdf/1606.01299.pdf) algorithm, with pre-trained filters to support 1.5 and 2x up-scaling, with options for low-res, high-res, and denoising.  Details of this algorithm can be found in our joint paper with AWS presented at [Mile High Video 2024](https://dl.acm.org/doi/10.1145/3638036.3640290).
+  - **TSENet** (Temporally Stabilized ETDS Network) – Optimized implementation of the [ETDS](https://github.com/ECNUSR/ETDS) algorithm for 2x up-scaling, with enhancements for multi-frame temporal stabilization.  Details of this algorithm can be found in our papers presented at Mile High Video 2025, and at the [NAB 2025 BEIT](https://nabpilot.org/product/tsenet-video-super-resolution-for-broadcast-television/) conference.
+  - **Enhanced EDSR** – Optimized implementation of the [EDSR](https://arxiv.org/pdf/1707.02921.pdf) algorithm for 2x up-scaling, using OpenVINO.
+  - **Enhanced BasicVSR** – Optimized implementation of the [BasicVSR](https://arxiv.org/pdf/2012.02181.pdf) algorithm for 2x up-scaling, using OpenVINO.
```

Comment:
It is considered not a good practice, to add links directly pointing to binary resources without landing page in the middle. I would advise to link pdf directly only as an additional option to the main link pointing to the landing page of desired paper:

```suggestion
  - **Enhanced BasicVSR** – Optimized implementation of the [BasicVSR](https://arxiv.org/abs/2012.02181) ([pdf](https://arxiv.org/pdf/2012.02181.pdf)) algorithm for 2x up-scaling, using OpenVINO.
```

It would then look like this:
  - **Enhanced BasicVSR** – Optimized implementation of the [BasicVSR](https://arxiv.org/abs/2012.02181) ([pdf](https://arxiv.org/pdf/2012.02181.pdf)) algorithm for 2x up-scaling, using OpenVINO.

Reply:
I've done a rebase of the commit/pr head on top of main - as there were important speed-up implemented. Please have an update before continuing @kevinacone